extensions/dbgobject/core/dbgobject.js

```diff
@@ -856,9 +856,7 @@ Loader.OnLoad(function() {
                 return getRestOfString(that, "", /*suggestedChunkSize*/64);
             } else {
                 return that.vals(length)
-                .then(function (chars) {
-                    return chars.map(function (c) { return String.fromCharCode(c); }).join("");
-                });
+                .then(chars => String.fromCharCode(...chars));
```

Comment:
That would be even better, except it looks like "RangeError: Maximum call stack size exceeded" gets thrown above a certain point?

Like I said in the other thread, if you've convinced yourself there's effectively no point at which string concatenation explodes and want to change it, go for it.

Reply:
Yeah, just noticed that too. That's a no go! FWIW, even if we encounter 1M+ characters in a single string (which is pretty unlikely - even the HTML5 spec doesn't seem to have strings that are that big), .reduce shouldn't explode. I don't think this is a big deal though, so reverting to what it was!